FindOpenGLES.cmake

```diff
@@ -9,46 +9,60 @@
 # OPENGLES_INCLUDE_DIRS - the OpenGLES include directory
 # OPENGLES_LIBRARIES - the OpenGLES libraries
 # OPENGLES_DEFINITIONS - the OpenGLES definitions
+#
+# Note:
+# On Windows with angle the *_INCLUDE_DIRS and
+# *_DEFINITIONS are undefined, but are set
+# global by the kodi-angle package.
 
-if(CORE_PLATFORM_NAME_LC STREQUAL rbpi)
-    set(_brcmprefix brcm)
-endif()
+if(WIN32)
+  find_package(kodi-angle REQUIRED)
+  set(OPENGLES_LIBRARIES kodi::angle::libGLESv2
+                         kodi::angle::libEGL)
+  set(OPENGLES_FOUND ${kodi-angle_FOUND})
```

Comment:
This is wrong, use `find_package_handle_standard_args`.

Reply:
Have you an example, has tried now and also on begin and not becomed it working with `find_package_handle_standard_args`